images/plbase/python-requirements.txt

```diff
@@ -1,6 +1,5 @@
 chevron==0.14.0
 flake8==5.0.4
-flake8-quotes==3.3.1
```

Comment:
Does this need to be removed?

Reply:
Discussed in some other PRs, but this is inconsistent with the black autoformatter, which we eventually want to run over the whole codebase (see https://github.com/PrairieLearn/PrairieLearn/issues/5027). Unfortunately that means a developer can't run black while this is installed (and I used this to format the code in the element file that I modified, which made development much easier).

Reply:
Ah, gotcha, makes sense.  I think we should avoid making unrelated changes in PRs (though, cc @nwalters512 for opinion here)

Reply:
Agreed, I only made this change because I personally find it a convenient tool to use.